### PR TITLE
File parsers

### PR DESCRIPTION
**Pull request by demesup**
image
image

### CODE REVIEW

**Comment by demesup** [comment]
image

File: src/main/java/org/transactionmanager/filemanager/ParserType.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package org.transactionmanager.filemanager;
+
+import lombok.Getter;
+
+@Getter
+public enum ParserType {
+    YAML("yaml"),
```
Comment (kolodgeeva) [inline src/main/java/org/transactionmanager/filemanager/ParserType.java:7]: why everything with dot and yaml without?

**Comment by kolodgeeva** [comment]
You have a great solution. I see a lot of good things, the map for action, and OOP.
The repository is not well designed. But we cover this topic later.
Remember that the repository is responsible for storing/fetching data from storage (file, db)
The controller and View are responsible for getting data from the user and communicating with the service/repository.
Don't worry, we handle this topic.

**Comment by kolodgeeva** [comment]
you can merge the changes and focus on the next topic

**Pull request merged**